frontend/src/locales/de.json

```diff
@@ -37,9 +37,9 @@
     "months": "Monate",
     "noDecay": "Keine Vergänglichkeit",
     "past_time": "Vergangene Zeit",
-    "received": "Empfangen",
+    "receive": "Empfangen",
```

Comment:
??? why not past tense?

Reply:
`<b-col cols="6" class="text-right">{{ $t(`decay.${typeId.toLowerCase()}`) }}</b-col>`